packages/apps/composer-app/src/constants.ts

```diff
@@ -11,7 +11,9 @@ export const INITIAL_CONTENT = [
   `# Welcome to Composer by DXOS
 ![img](https://dxos.org/images/logo/dxos-logotype-white.svg)
 ### Disclaimer
-Composer is a proving ground for the [DXOS framework](https://docs.dxos.org/) and our evolving plugin ecosystem. While Composer is stable and robust, it should not be treated as production-ready software quite yet.
+Composer is currently in beta, offering an early preview of its collaborative and programmable features; we would greatly appreciate your feedback to help improve it before the official launch.
```

Comment:
Composer is currently in Beta. We are offering an early preview of its collaborative and programmable features, and we would greatly appreciate your feedback to help improve it before the official launch.